src/chat/discord.md

```diff
@@ -1,4 +1,104 @@
 ## Discord
 
-We haven't yet written the documentation for Discord, but if you'd like to help
-out, this is the place for it to go!
+Rust's Discord is used by Community, Lang, Ops, and Documentation teams, as well as their working groups. It is also the home of Domain Working Groups and provides a space for general discussion among Rust users, contributors, and beginners. 
+
+# Where to go for help with using Discord
+Discord's support center provides documentation about its [user interface](https://support.discordapp.com/hc/en-us/categories/200404398) and [account settings](https://support.discordapp.com/hc/en-us/categories/200404358).
+
+# Getting started
+
+1) Understand community standards <br/>
+Before joining the conversation on Discord, you can prepare by reading the [Code of Conduct and Moderation Guidelines](https://www.rust-lang.org/community#conduct).
```

Comment:
Should add Discord's Community Guidelines as well. https://discordapp.com/guidelines